spec/rubocop/cop/style/map_into_array_spec.rb

```diff
@@ -215,6 +311,36 @@
     RUBY
   end
 
+  it 'does not register an offense when pushing anonymously-forwarded block', :ruby31 do
+    expect_no_offenses(<<~RUBY)
+      def foo(&)
+        dest = []
+        src.each { |e| dest.push(&) }
+        do_something
```

Comment:
Does this `do_something` have any meaning? If not, it can probably be removed.

Reply:
Good point, that was only relevant for autocorrection (to create an empty context.)